exercises/flatten-array/example.sml

```diff
@@ -1,5 +1,5 @@
 datatype 'a nestedList =
-	  Leaf of 'a			     (* leaf *)
+	  Leaf of 'a			           (* leaf *)
```

Comment:
Definitely, this makes more sense. I think I probably modified this example from another piece of code that was for a tree and left the Leaf constructor. I'll update.